src/tsconfig/rulesets/strict.ts

```diff
@@ -0,0 +1,27 @@
+import { configurableViaJsiiSettings } from './generated';
+import { jsiiIncompatibleOptions } from './minimal';
```

Comment:
Can't we put the reusable parts into a separate files? It's confusing me a bit that the reusable bits are in the same files as *some* user-facing `RuleSets`. It seems a bit arbitrary what they're mixed with.

Reply:
Done!